mmseg/datasets/pipelines/transforms.py

```diff
@@ -1012,6 +1014,9 @@ def __call__(self, results):
             y2 = np.clip(y1 + cutout_h, 0, h)
             results['img'][y1:y2, x1:x2, :] = self.fill_in
 
+            for key in results.get('seg_fields', []):
+                results[key][y1:y2, x1:x2] = self.ignore_index
```

Comment:
If `seg_fill_in` is None, skip this modification.

Reply:
> If `seg_fill_in` is None, skip this modification.

like `if seg_fill_in == True:`?